lib/database/bootstrapper_runtime/handler.py

```diff
@@ -235,35 +229,25 @@ def handler(event, context):
                 print(f"Running migrations to PgSTAC {params['pgstac_version']}...")
                 Migrate(pgdb).run_migration(params["pgstac_version"])
 
+        with psycopg.connect(
+            eoapi_db_admin_conninfo,
+            autocommit=True,
+            options="-c search_path=pgstac,public -c application_name=pgstac",
+        ) as conn:
+            print("Customize PgSTAC database...")
             # Update permissions to eoAPI user to assume pgstac_* roles
             with conn.cursor() as cur:
-                print(f"Update *{eoapi_params['username']}* permissions...")
+                print(f"Update '{eoapi_params['username']}' permissions...")
                 update_user_permissions(
                     cursor=cur,
                     db_name=eoapi_params["dbname"],
                     username=eoapi_params["username"],
                 )
```

Comment:
we need to have the search path set to pgstac,public in order to update the user permission or granting `pgstac_*`  to user won't work 😅

Reply:
One trick that might be useful is to just set the search_path on the new database you are creating.
`ALTER DATABASE mydb SET SEARCH_PATH TO pgstac, public;`